doc/source/regression/dwt-idwt.md

```diff
@@ -65,7 +42,7 @@ cD
 
 ## Inverse Discrete Wavelet Transform
 
-Now, let's do the opposite operation: an Inverse Discrete Wavelet Transform:
```

Comment:
@agriyakhetarpal I saw that you added these commas, but I think most English style guides advise against a comma in this situation. The sentence is perfectly unambiguous without the comma; the comma just weighs it down.

Reply:
I see, thanks. I think commas do fit if it is an "Introductory word", based on these guidelines: https://www.bachelorprint.com/commas/comma-after-now/.

That said, I find that the added emphasis might fit elsewhere but is not this sentence, especially because it's short in length. We should drop it, indeed.